components/DonationListItem.js

```diff
@@ -1,63 +1,78 @@
 import styled from "styled-components";
 import { useState } from "react";
 
-export default function DonationListItem({ donation }) {
-  const [toggle, setToggle] = useState(false);
+export default function DonationListItem({ donation, myUserID, handleDelete }) {
+  const [toggleDonationDetails, setToggleDonationDetails] = useState(false);
   function handleClick() {
-    setToggle(!toggle);
+    setToggleDonationDetails(!toggleDonationDetails);
   }
-  if (toggle) {
+  if (toggleDonationDetails) {
     return (
       <StyledCard>
         <p>{donation.image}</p>
         <p>Kategorie: {donation.category}</p>
         <p>Beschreibung: {donation.description}</p>
-        <p>Entfernung: {donation.distance}</p>
-        <p>Haltbar bis: {donation.good_before}</p>
-        <p>Kontakt: {donation.contact_information}</p>
+        <p>Haltbar bis: {donation.bestBefore}</p>
+        <p>PLZ: {donation.zipCode}</p>
+        <p>Stadtteil: Hamburg- {donation.district}</p>
+        <p>Kontakt: {donation.contactInformation}</p>
         <label for="checkbox1">Bio</label>
         <input
           type="checkbox"
           id="checkbox1"
           name="checkbox1"
-          value="value1"
           checked={donation.isBio}
+          readOnly
         />
         <label for="checkbox2">Vegetarisch</label>
         <input
           type="checkbox"
           id="checkbox2"
           name="checkbox2"
-          value="value2"
           checked={donation.isVegetarian}
+          readOnly
         />
         <label for="checkbox3">Vegan</label>
         <input
           type="checkbox"
           id="checkbox3"
           name="checkbox3"
-          value="value3"
           checked={donation.isVegan}
+          readOnly
         />
         <label for="checkbox4">Glutenfrei</label>
         <input
           type="checkbox"
           id="checkbox4"
           name="checkbox4"
-          value="value4"
           checked={donation.isGlutenfree}
+          readOnly
         />
         <StyledLessInfoButton type="button" onClick={() => handleClick()}>
           Weniger Infos
         </StyledLessInfoButton>
+        {
+          (donation.userID = myUserID ? (
+            //editbutton in die blauen klammern
+            <StyledDeleteButton
+              type="button"
+              onClick={() => handleDelete(donation.id)}
+            >
+              Löschen
+            </StyledDeleteButton>
+          ) : (
+            ""
+          ))
+        }
```

Comment:
```suggestion
        {
          (donation.userID === myUserID && (
            //editbutton in die blauen klammern
            <StyledDeleteButton
              type="button"
              onClick={() => handleDelete(donation.id)}
            >
              Löschen
            </StyledDeleteButton>
          ))
        }
```

Reply:
great suggestion - thank you I changed it as requested :) and it saved some code :)